src/map/GMap.js

```diff
@@ -163,11 +163,16 @@
     GMap.prototype.publish("zoom", 4, "number", "Zoom Level", null, { tags: ["Basic"] });
 
     GMap.prototype.publish("panControl", true, "boolean", "Pan Controls", null, { tags: ["Basic"] });
-    GMap.prototype.publish("zoomControl", true, "boolean", "Pan Controls", null, { tags: ["Basic"] });
-    GMap.prototype.publish("mapTypeControl", false, "boolean", "Pan Controls", null, { tags: ["Basic"] });
-    GMap.prototype.publish("scaleControl", true, "boolean", "Pan Controls", null, { tags: ["Basic"] });
+    GMap.prototype.publish("zoomControl", true, "boolean", "Zoom Controls", null, { tags: ["Basic"] });
+    GMap.prototype.publish("fullscreenControl", true, "boolean", "Fullscreen Controls", null, { tags: ["Basic"] });
+    GMap.prototype.publish("mapTypeControl", false, "boolean", "Map Type Controls", null, { tags: ["Basic"] });
+    GMap.prototype.publish("scaleControl", true, "boolean", "Scale Controls", null, { tags: ["Basic"] });
     GMap.prototype.publish("streetViewControl", false, "boolean", "Pan Controls", null, { tags: ["Basic"] });
     GMap.prototype.publish("overviewMapControl", false, "boolean", "Pan Controls", null, { tags: ["Basic"] });
+    
+    GMap.prototype.publish("streetViewStreetAddress", '', "string", "Address to display via StreetView API", null, { tags: ["Basic"] });
+    
+    GMap.prototype.publish("streetViewTargetSelector", '', "string", "CSS selector for streetview target (defaults to widget target)", null, { tags: ["Basic"] });
```

Comment:
It takes a CSS selector and will render the Street View widget in the resulting selected element (instead of rendering in the current google maps widget).